### DISCUSSION TITLE
This site can’t be reached findlet.ng refused to connect.

### DISCUSSION THREAD

**Discussion by Wence17** [vercel/community#6750] [Help] [answered]
### Summary
@rauchg @tomocchino @TooTallNate @vvo 
After I added a custom domain and updated the dns records; the website was accessible after a few minutes and then can no longer be accessed after a few minutes.

I already checked if the domain has propagated across the globe and it's all positive.
the site can be accessed from this link provided by vercel [here](https://findlet-wence17-s-team.vercel.app/) but cannot be accessed from the added [domain](https;//findlet.ng)

Please what could be the problem??????

### Example

https://findlet-wence17-s-team.vercel.app/

### Steps to Reproduce

I don't really know what you might be needing. this is my link to the website [findlet.ng](https://findlet.ng) and my project on vercel is findlet.

if you need more clarity please specify. 

![image](https://github.com/vercel/community/assets/93437186/7c15b1d1-eaff-443f-81fa-44c7d69d2043)

**Comment by swarnava** [comment]
Hey,

are you accessing it from Nigeria? We have some known ISP block in Nigeria: https://github.com/orgs/vercel/discussions/3630.

**Comment by Wence17** [comment]
Yes I am using it from Nigeria

**Comment by Wence17** [comment]
What do you suggest I do?

**Reply by amyegan** [reply]
We are aware of the ISP block and working to resolve the issue. Until the block is removed, there is a workaround available. Step-by-step instructions can be found in https://github.com/vercel/community/discussions/3630#discussioncomment-8810118

**Comment by Wence17** [comment, marked as answer]
I used name server instead of dns record and it works pretty well

**Reply by Ethical-Saleem** [reply]
Hi. I am also having the same issue as yours. How were you able to correct it. I visited your site now and was able to access it.

**Comment by Wence17** [comment]
Ended up using nameserver instead of dns record.

**Reply by Ethical-Saleem** [reply]
Thanks. I also used nameserver and i haven't encountered the issue since
Still monitoring it though

**Comment by Wence17** [comment]
Glad it worked!
